dymos/utils/test/test_misc.py

```diff
@@ -14,6 +14,20 @@ def test_get_rate_units(self):
 
         self.assertEqual(rate2_units, 'm/s**2')
 
+    def test_rate_units_for_unitless(self):
+        rate_units = get_rate_units('unitless', 's', deriv=1)
+        self.assertEqual(rate_units, '1/s')
+        rate2_units = get_rate_units('unitless', 's', deriv=2)
+        self.assertEqual(rate2_units, '1/s**2')
+
+    def test_rate_units_for_unitless_time(self):
+        for time_units in (None, 'unitless'):
```

Comment:
unitless time is kind of weird, but I guess it is a valid corner case. Maybe more like non-dimensionalized time. I guess if a user wants to see the differentiation status for the rate*, they could just define a `unitless_time` unit, so you get 'unitless/unitless_time**2' in the plots.

Reply:
Yeah I don't really expect this to come up ever, but figured I'd deal with it while I was here.